### ISSUE TITLE
RSS Feed for the Blog

### ISSUE THREAD

**Issue by realeinherjar** [fish-shell/fish-site#115]
Would be great to have a RSS feed for the Blog: https://fishshell.com/blog/index.html

**Comment by ridiculousfish** [comment]
Good call, probably doable with GitHub pages. I've got plans to write some actual blog posts.

**Comment by zanchey** [comment]
Fixed with e99f26a4d83a
